records_mover/db/__init__.py

```diff
@@ -1,3 +1,9 @@
+__all__ = [
+    'DBDriver',
+    'LoadError',
+    'create_sqlalchemy_url',
+]
+
 from .driver import DBDriver  # noqa
-from .factory import db_driver  # noqa
 from .errors import LoadError  # noqa
+from .connect import create_sqlalchemy_url
```

Comment:
I renamed `create_db_url()` to make the name a little more specific and am now exporting it here because of an internal use.